lib/middleware.js

```diff
@@ -323,15 +323,15 @@ exports.mwl = function(req, rsp, next){
   // `.js` (Browserify) are actually being used to specify
   // source files
 
-  if (['js'].indexOf(ext) === -1) {
```

Comment:
nope. my guess would be the code evolved into that from something else. good catch. :)